src/test/__test__/edit_update_complet.test.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] Disabling linters is not a good practice in the developers' world. Linters are put in place to help us write good quality code. Kindly fix them instead 🔫